### ISSUE TITLE
Does KeepKey support GPG?

### ISSUE THREAD

**Issue by doolio** [romanz/trezor-agent#440]
https://github.com/romanz/trezor-agent/blob/3c911e99a0394278104564092225d67c75e74b99/agents/keepkey/setup.py#L7

Suggests it does.

https://github.com/romanz/trezor-agent/blob/3c911e99a0394278104564092225d67c75e74b99/agents/keepkey/setup.py#L35-L37

Yet, there appears to be no GPG entry points defined.

https://github.com/romanz/trezor-agent/blob/3c911e99a0394278104564092225d67c75e74b99/agents/keepkey/keepkey_agent.py#L1-L5

L1 imports the GPG module but there are then no `gpg_tool` or `gpg_agent` lambda function assignments like we have for the other hardware devices agents.

https://github.com/romanz/trezor-agent/blob/3c911e99a0394278104564092225d67c75e74b99/libagent/device/keepkey.py#L10

Suggests only SSH is supported.

**Comment by SlugFiller** [comment]
If you have one, you can simply enable the agent and test.

**Comment by doolio** [comment]
I only have a Trezor T.

**Comment by romanz** [comment]
@keepkeyjon IIUC, GPG wasn't supported initially on Keepkey - and was added only in https://github.com/keepkey/keepkey-firmware/commit/95e41aa3bd532267490a145bc0fc1c8deb436b16, right?

**Comment by romanz** [comment]
FTR, Keepkey support was added in https://github.com/romanz/trezor-agent/commit/946ab633d493293f80f8c7a1d04f5211d4007ea7#diff-23da18a1a68813740f5fda9f691ebbe1f1bc4f411cf7762cc2caa87fa3b04c1f, when it didn't support GPG.

**Comment by doolio** [comment]
That would explain the current state. Is it a matter of simply adding the entry points and the lambda functions I discuss above? If so I can submit such a PR for consideration.

**Comment by romanz** [comment]
> Is it a matter of simply adding the entry points and the lambda functions I discuss above?

Not sure - it should also be tested with the latest Keepkey firmware, to make sure it indeed works.

**Comment by doolio** [comment]
Hmm ok. I only have a trezor T so can't help with such tests.

**Comment by SlugFiller** [comment]
The best motivation for contribution is needing it, or knowing someone who needs it. Worrying about support for a device you don't have and don't know anyone who has is noble, but most often not particularly productive.

**Comment by doolio** [comment]
You're right of course. I'm just highlighting potential issues where I see them.

**Comment by romanz** [comment]
> @keepkeyjon IIUC, GPG wasn't supported initially on Keepkey - and was added only in [keepkey/keepkey-firmware@95e41aa](https://github.com/keepkey/keepkey-firmware/commit/95e41aa3bd532267490a145bc0fc1c8deb436b16) (#8), right?

CC: @keepkey @pastaghost @markrypto @markrypt0 @mrnerdhair @bgok
